apps/rps-game/src/contexts/UserContext.tsx

```diff
@@ -88,7 +88,7 @@ export default function UserContextProvider({ children }: Props) {
         ADD_AS_CHILD_MULTISIGN,
         (arg: any, t: any) => [
           arg(factoryAndFilterAddress, t.Address),
-          arg(factoryAndFilterAddress, t.String),
+          arg(factoryAndFilterAddress, t.Address),
```

Comment:
Does this need arg need to be duplicated?

Reply:
Good Q, the transaction signature is`(factoryAddress: Address, filterAddress: Address)` which in the general case don't have to be the same address. But filter and factory happen to be the same address in this case, so the address needs to be passed for both args.